### PR TITLE
Return error if creating of clients fails in framework

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

### CODE REVIEW

File: tests/framework/framework.go [hunk 1]
```diff
@@ -132,14 +132,14 @@ func NewFramework(prefix string, config Config) (*Framework, error) {
 	// clients
 	kcs, err := f.GetKubeClient()
 	if err != nil {
-		err = errors.Wrap(err, "ERROR, unable to create K8SClient")
+		return nil, errors.Wrap(err, "ERROR, unable to create K8SClient")
```
Comment (jeffvance) [inline tests/framework/framework.go]: don't need else since you've returned above.

File: tests/framework/framework.go [hunk 2]
```diff
@@ -132,14 +132,14 @@ func NewFramework(prefix string, config Config) (*Framework, error) {
...
 	} else {
 		f.K8sClient = kcs
 	}
 
 	cs, err := f.GetCdiClient()
 	if err != nil {
-		err = errors.Wrap(err, "ERROR, unable to create CdiClient")
+		return nil, errors.Wrap(err, "ERROR, unable to create CdiClient")
```
Comment (jeffvance) [inline tests/framework/framework.go]: ditto

**Comment by jeffvance** [comment]
needs rebase

File: tests/framework/framework.go (hunk 2 above)
Comment (jeffvance) [reply tests/framework/framework.go]: I'm sure you're busy... my re-review today shows no code change and no comment from you. Will wait to merge.

File: tests/framework/framework.go (hunk 1 above)
Comment (awels) [reply tests/framework/framework.go]: You are right, removing else.

File: tests/framework/framework.go (hunk 2 above)
Comment (awels) [reply tests/framework/framework.go]: I was on PTO Tuesday, that is why I didn't answer.

**Comment by jeffvance** [comment]
lgtm
Waitting for ci then will merge.

**Pull request merged**